src/Services/MASA.PM.Service.Admin/Program.cs

```diff
@@ -33,13 +30,39 @@
 }
 #endregion
 
+if (builder.Environment.IsProduction())
+{
+    //var daprClient = GetDaprClient(builder.Services);
```

Comment:
If you don't need it anymore, you can remove it.  if you just comment it to test, you should add comment tell me.